### PR TITLE
Added german Translation

### PR DESCRIPTION
**Pull request by ProfDrLuigi**

### CODE REVIEW

File: HeliPort/Appearance/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+"No Status Information Avaliable" = "Keine Status Informationen verfügbar";
+"None" = "Keine";
+"Open Network Preferences..." = "Netzwerk Einstellungen öffnen...";
+"Open Wireless Diagnostics..." = "Diagnose für Drahtlose Umgebungen öffnen...";
+"Password:" = "Passwort:";
+"Password" = "Passwort";
+"Quit HeliPort" = "HeliPort beenden";
+"Reachable" = "Erreichbar";
+"Remember this network" = "Dieses Netzwerk speichern";
+"Security:" = "Verschlüsselung:";
```
Comment (Goshin) [inline HeliPort/Appearance/de.lproj/Localizable.strings:27]: Sicherheit

File: HeliPort/Appearance/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+"WPA2 Personal" = "WPA2 Personal";
+"WPA2/WPA3 Enterprise" = "WPA2/WPA3 Enterprise";
+"WPA2/WPA3 Personal" = "WPA2/WPA3 Personal";
+"WPA3 Enterprise" = "WPA3 Enterprise";
+"WPA3 Personal" = "WPA3 Personal";
+"Wi-Fi: Connected" = "WLAN: Verbunden";
+"Wi-Fi: Connecting" = "WLAN: Verbinden";
+"Wi-Fi: Looking for Networks..." = "WLAN: Suche nach Netzwerken...";
+"Wi-Fi Network \"" = "WLAN Netzwerk \"";
+"Wi-Fi: Off" = "WLAN: Deaktiviert";
```
Comment (Goshin) [inline HeliPort/Appearance/de.lproj/Localizable.strings:49]: WLAN: aus

File: HeliPort/Appearance/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+"Wi-Fi: On" = "WLAN: Aktiviert";
```
Comment (Goshin) [inline HeliPort/Appearance/de.lproj/Localizable.strings:50]: WLAN: ein

Comment (ProfDrLuigi) [reply HeliPort/Appearance/de.lproj/Localizable.strings]: Done

File: HeliPort.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -64,14 +64,15 @@
...
 		50B86ABD24B22FEE008E4FE4 /* pt-BR */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = "pt-BR"; path = "pt-BR.lproj/Localizable.strings"; sourceTree = "<group>"; };
 		6A54F80824B24B9700B8D651 /* it */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = it; path = it.lproj/Localizable.strings; sourceTree = "<group>"; };
 		6A78687824B26AF6007832EE /* tr */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = tr; path = tr.lproj/Localizable.strings; sourceTree = "<group>"; };
-		42979EFF24B3A27200F15FBD /* el */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = el; path = el.lproj/Localizable.strings; sourceTree = "<group>"; };
 		7557CDF324935260000F0F71 /* Credits.rtf */ = {isa = PBXFileReference; lastKnownFileType = text.rtf; path = Credits.rtf; sourceTree = "<group>"; };
```
Comment (williambj1) [inline HeliPort.xcodeproj/project.pbxproj:76]: please restore this line

File: HeliPort.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -64,14 +64,15 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		2A66531D24B6E172002F88EE /* de */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = de; path = de.lproj/Localizable.strings; sourceTree = "<group>"; };
```
Comment (williambj1) [inline HeliPort.xcodeproj/project.pbxproj:67]: Please move this line lower

File: HeliPort.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -64,14 +64,15 @@
...
 		2B879D3A8D737E5EF06C350F /* Pods-HeliPort.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-HeliPort.release.xcconfig"; path = "Target Support Files/Pods-HeliPort/Pods-HeliPort.release.xcconfig"; sourceTree = "<group>"; };
+		42979EFF24B3A27200F15FBD /* el */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = el; path = el.lproj/Localizable.strings; sourceTree = "<group>"; };
```
Comment (williambj1) [inline HeliPort.xcodeproj/project.pbxproj:69]: Please restore this line

**Pull request merged**